modules/gltf/gltf_document.cpp

```diff
@@ -505,25 +505,8 @@ String GLTFDocument::_gen_unique_animation_name(Ref<GLTFState> state, const Stri
 	return name;
 }
 
-String GLTFDocument::_sanitize_bone_name(const String &name) {
-	String p_name = name.camelcase_to_underscore(true);
-
-	RegEx pattern_nocolon(":");
-	p_name = pattern_nocolon.sub(p_name, "_", true);
-
-	RegEx pattern_noslash("/");
-	p_name = pattern_noslash.sub(p_name, "_", true);
-
-	RegEx pattern_nospace(" +");
-	p_name = pattern_nospace.sub(p_name, "_", true);
-
-	RegEx pattern_multiple("_+");
-	p_name = pattern_multiple.sub(p_name, "_", true);
-
-	RegEx pattern_padded("0+(\\d+)");
-	p_name = pattern_padded.sub(p_name, "$1", true);
```

Comment:
": /" are already banned.

` ` Spaces are valid.

`_`  Multiple underscores is valid.

Regex "0+(\\d+)" appears to be a way hide injected nodes.

Reply:
". : @ / \""

". @ \" I'm pretty sure these are all valid skeleton characters.

Reply:
". : @ / "" are valid Skeleton bone names.

![image](https://user-images.githubusercontent.com/32321/111371129-3fc33800-8656-11eb-8c5c-f01d4fbc89bf.png)

Reply:
![image](https://user-images.githubusercontent.com/32321/111371639-ee677880-8656-11eb-8047-700bee656e54.png)

The minimum appears to be stripping ": /"